lib/actions/appleMusic.js

```diff
@@ -155,15 +145,16 @@ function appleMusic(player, values) {
                 return player.coordinator.clearQueue()
                   .then(() => player.coordinator.addURIToQueue(queueTracks[0].uri, queueTracks[0].metadata, true, 1))
                   .then(() => {
-                      if (radioOn(player)) {
-                        return promise.then(() => player.coordinator.setAVTransport(queueURI,''));
+                      if (isRadioOrLineIn(player.coordinator.state.currentTrack.uri)) {
```

Comment:
You should check `player.coordinator.avTransportUri`, the currentTrack will be the actual uri streaming from I believe, and might differ.